### PR TITLE
Prometrics config and Help implementation

### PR DESCRIPTION
**Pull request by Buzzglo**
…ommand to /commands and added metric hook to monitor

### CODE REVIEW

File: src/gluon/commands/help/Help.ts [hunk 1]
```diff
@@ -79,12 +80,15 @@ export class Help implements HandleCommand<HandlerResult> {
             if (this.selectedOption === undefined) {
                 return await this.displayCategories(ctx);
             } else if (this.selectedOption.startsWith(QMConfig.subatomic.commandPrefix)) {
+                this.succeedCommand();
                 return this.displayCommandToBeRun(ctx);
             } else {
+                this.failCommand();
```
Comment (kbristow) [inline src/gluon/commands/help/Help.ts]: I dont think this is a fail. Check with @kbester

Comment (KieranHons) [inline src/gluon/commands/help/Help.ts]:
```suggestion
                this. succeedCommand();
```

Comment (KieranHons) [reply src/gluon/commands/help/Help.ts]: This is still a successful run, its the second phase of the help. i.e after they've chosen a category

Comment (KieranHons) [reply src/gluon/commands/help/Help.ts]: @KieranHons

Comment (KieranHons) [reply src/gluon/commands/help/Help.ts]: Ah thats pretty cool

Comment (Buzzglo) [reply src/gluon/commands/help/Help.ts]: Changed thanks

**Pull request merged**